Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -522,6 +529,18 @@ private void ContentGrid_ManipulationDelta(object sender, ManipulationDeltaRoute
             SwipeStatus = newSwipeStatus;
         }
 
+        private void ContentStoryboard_Completed(object sender, object e)
```

Comment:
Now that’s fixed. But what are the benefits of doing this? Shouldn’t other events also be unsubscribed when the control is unloaded?